Dockerfile

```diff
@@ -7,7 +7,11 @@ RUN apk add --no-cache bash \
         bzip2-dev \
         gcc \
         libc-dev \
-    && pip install --no-cache-dir gevent==1.1.2 flask==0.11.1 kafka_python==1.3.1 requests==2.10.0 cloudant==2.1.0
+        py-pip \
+        python-dev \
```

Comment:
Seems like py-pip and python-dev are not actually needed for this. At least, it seems to work when I remove them.